src/components/WorldMap/index.js

```diff
@@ -139,18 +141,39 @@ class WorldMap extends Component {
         return crimeA.distanceToUser - crimeB.distanceToUser;
       });
 
-      warCrimes[0].classList.add('closest');
+      warCrimes.forEach(crime => {
+        crime.classList.add('closest');
+        crime.style.animationPlayState = 'paused';
```

Comment:
animation-play-state is een css attribute die je ook by default op paused kan zetten